sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_generated/v3_1/aio/_text_analytics_client.py

```diff
@@ -57,6 +57,7 @@ async def _send_request(self, http_request: HttpRequest, **kwargs: Any) -> Async
         :rtype: ~azure.core.pipeline.transport.AsyncHttpResponse
         """
         path_format_arguments = {
+            'ApiVersion': self._serialize.url("api_version", api_version, 'str', skip_quote=True),
```

Comment:
Not sure if this is handling the new ApiVersion parameterizing in the swagger correctly - `api_version` does not exist in this scope.

Reply:
swagger transform fixes this for now